<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PDF工具</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/knighthood/OneDrive/桌面/熊二表情包/熊二表情包41.jpg</normaloff>C:/Users/knighthood/OneDrive/桌面/熊二表情包/熊二表情包41.jpg</iconset>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="word">
     <attribute name="icon">
      <iconset>
       <normaloff>../pyqt5页面/images/PDF2Word.png</normaloff>../pyqt5页面/images/PDF2Word.png</iconset>
     </attribute>
     <attribute name="title">
      <string>word</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget2_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>871</width>
        <height>561</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="word2pdf">
       <attribute name="title">
        <string>word转pdf</string>
       </attribute>
       <widget class="QLabel" name="label1_1_1">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>72</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>word路径</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn1_1_1">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件（夹）</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser1_1_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>256</width>
          <height>61</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label1_1_3">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>30</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>相关信息展示</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn1_1_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>350</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>开始转换</string>
        </property>
       </widget>
       <widget class="QLabel" name="label1_1_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>72</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>保存路径</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser1_1_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>220</y>
          <width>256</width>
          <height>61</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn1_1_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>160</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件夹</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser1_1_3">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>80</y>
          <width>321</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="none">
       <attribute name="title">
        <string>待开发</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="pdf">
     <attribute name="icon">
      <iconset>
       <normaloff>../pyqt5页面/images/PDF2Img.png</normaloff>../pyqt5页面/images/PDF2Img.png</iconset>
     </attribute>
     <attribute name="title">
      <string>pdf</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget2_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>541</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="pdf2word">
       <attribute name="title">
        <string>pdf转word</string>
       </attribute>
       <widget class="QPushButton" name="btn2_1_1">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>0</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件（夹）</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn2_1_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>120</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件（夹）</string>
        </property>
       </widget>
       <widget class="QLabel" name="label2_1_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>72</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>保存路径</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn2_1_3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>340</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>开始转换</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser2_1_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>331</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label2_1_1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>0</y>
          <width>72</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF路径</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser2_1_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>331</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label2_1_3">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>20</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>相关信息展示</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser2_1_3">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>50</y>
          <width>256</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pdf2img">
       <attribute name="title">
        <string>pdf转img</string>
       </attribute>
       <widget class="QPushButton" name="btn2_2_1">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件（夹）</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn2_2_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件（夹）</string>
        </property>
       </widget>
       <widget class="QLabel" name="label2_2_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>72</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>保存路径</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn2_2_3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>310</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>开始转换</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser2_2_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>331</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label2_2_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>72</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF路径</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser2_2_1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>331</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label2_2_3">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>30</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>相关信息展示</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser2_2_3">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>60</y>
          <width>256</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pdf2text">
       <attribute name="title">
        <string>pdf提取文字</string>
       </attribute>
       <widget class="QPushButton" name="btn2_3_1">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件（夹）</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn2_3_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>280</y>
          <width>121</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>开始转换</string>
        </property>
       </widget>
       <widget class="QLabel" name="label2_3_1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>72</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF路径</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser2_3_1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>331</width>
          <height>111</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label2_3_3">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>10</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>相关信息展示</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit2_3_1">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>210</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label2_3_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>210</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>选择页码</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser2_3_4">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>10</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label2_3_5">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>10</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>总页数：</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit2_3_1">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>60</y>
          <width>501</width>
          <height>431</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit2_3_2">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>10</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="PDF2Text_2">
     <attribute name="icon">
      <iconset>
       <normaloff>../pyqt5页面/images/PDF2Text.png</normaloff>../pyqt5页面/images/PDF2Text.png</iconset>
     </attribute>
     <attribute name="title">
      <string>PDF提取文字</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser3_1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label2_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>72</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF路径</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn3_1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>170</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>相关信息展示</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>选择页码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit3_1">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn3_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>110</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>开始提取</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit3_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>561</width>
        <height>271</height>
       </rect>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/pdf.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_background</zorder>
   <zorder>tabWidget1</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionVersion"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="PDF2Word"/>
    <addaction name="PDF2Img"/>
    <addaction name="PDF2Text"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="PDF2Word">
   <property name="icon">
    <iconset>
     <normaloff>../pyqt5页面/images/PDF2Word.png</normaloff>../pyqt5页面/images/PDF2Word.png</iconset>
   </property>
   <property name="text">
    <string>PDF转word</string>
   </property>
  </action>
  <action name="PDF2Img">
   <property name="icon">
    <iconset>
     <normaloff>../pyqt5页面/images/PDF2Img.png</normaloff>../pyqt5页面/images/PDF2Img.png</iconset>
   </property>
   <property name="text">
    <string>PDF转图片</string>
   </property>
  </action>
  <action name="PDF2Text">
   <property name="icon">
    <iconset>
     <normaloff>../pyqt5页面/images/PDF2Text.png</normaloff>../pyqt5页面/images/PDF2Text.png</iconset>
   </property>
   <property name="text">
    <string>PDF提取文字</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>版本号</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
